README.md

```diff
@@ -38,16 +38,11 @@
 
```

Comment:
Great job @ifeanyi305 🤝

- [ ] [Optional] Kindly remove the second author from your readme file since only one person contributed to this pull request. 
- [ ] [Optional] You may also fix the social handle links so that they work as they are intended to work. i.e, redirect the user to the author's GitHub account.